.gitignore

```diff
@@ -0,0 +1,162 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
this looks like it is copied from one of our other REPOs. there isn't really a need for this.